README.md

```diff
@@ -84,7 +84,7 @@ With each release a new branch is created, for example the `92.0.260` release co
 If you're new to `CefSharp` and are downloading the source to check it out, please use a **Release** branch.
 
 **&ast;** VC++ 2022 is required starting with version 138<br/>
-**&ast;&ast;** For .Net Core Packages 6 or greater is required. 
+**&ast;&ast;** For .Net Packages 6 or greater is required. 
```

Comment:
I wondering if changing this to say `For NETCore Packages .Net 6 or greater is required` might give some added clarity